ios/.sourcery.yml

```diff
@@ -1,9 +1,16 @@
-sources:
-  - KMMIC/Sources
-templates:
-  - tools/sourcery/templates
-output:
-  KMMICTests/Sources/Mocks/Sourcery 
-args:
-  autoMockableTestableImports:
-    - KMMIC
+configurations:
```

Comment:
It seems you separated into 2 files, what is the approach?

Reply:
We need to wait for the generated code from the Shared module to complete then we can run the normal Sourcery to generate the mock classes